book/src/appendices/b_migration_notes/cgmath_to_nalgebra.md

```diff
@@ -0,0 +1,228 @@
+# CgMath to Nalgebra
+
+## How To Use
+
+This cheat sheet is split up into the following sections:
+
+* **Type Changes:** Previously you used `this::Type`, now you use `another::Thing`
+* **Logic Changes:** Previously you had `object.method(args)`, now you use `object.other(stuff)`
+
+Most changes will have accompanying explanations and code examples on how to switch.
+
+This document is by no means exhaustive, so if there is something missing, or if you can clarify any changes, please correct this!
+
+The text is designed to be searchable, so if you are looking for a specific type or method, please use Ctrl + F: `TypeName`. If you cannot find it in the document, likely we missed it during writing. Please send us a pull request!
+
+## Type Changes
+
+Many types retain the same type name, just under the `nalgebra` namespace:
+
+```patch
+-use amethyst::core::nalgebra::{Vector2, Vector3, Matrix4};
```

Comment:
Could we reverse this, so + is on the things we're moving to?

Reply:
woops, manually created patch :sweat_smile: